### ISSUE TITLE
Don't fail when response is `nil`

### ISSUE THREAD

**Issue by rngtng** [aserafin/grape_logging#19]
As of https://github.com/ruby-grape/grape/pull/1240 the `#after` hook is called on even when exception is raised. This breaks grape_logging which assumes `response`, (`@app_response`) to be set see: https://github.com/aserafin/grape_logging/blob/master/lib/grape_logging/middleware/request_logger.rb#L36

See https://github.com/ruby-grape/grape/issues/1265 for discussion

**Comment by rngtng** [comment]
related: https://github.com/aserafin/grape_logging/pull/10

**Comment by guizmaii** [comment]
Do you have an idea on how to solve this ? I'm working on this problem in my fork.

**Comment by rngtng** [comment]
well just check on the value of `response`;

a dirty hack would be:
```ruby
response.try(:status)
```

or better
```ruby
status: (response.nil? ? nil : response.status) # or 'error' instead of nil
```

**Comment by guizmaii** [comment]
Ok thanks. It's done in my PR #16

**Comment by aserafin** [comment]
@rngtng should be fixed since I merged #16
